materials/other_data_structures_R.md

```diff
@@ -5,102 +5,109 @@ title: Other Data Structures
 language: R
 ---
 
-> Remind students to [Install Bioconductor](http://www.bioconductor.org/install/).
+> Remind students to [install Bioconductor](http://www.bioconductor.org/install/).
 
-> Remember to downloade [*E. coli* genome data]({{ site.baseurl }}/data/ecoli_coding.fasta)
+> Remember to download [Archaea genome data]({{ site.baseurl }}/data/archaea_dna.zip).
  
```

Comment:
@ethanwhite: Do you have an `ecoli_coding.fasta` file? How do you feel about using a data file pulled from the exercise (`Loops-multiple-files-R.md`)?